setup.py

```diff
@@ -35,6 +35,9 @@
         'console_scripts': [
             'dallinger = dallinger.command_line:dallinger',
         ],
+        'dallinger.experiments': [
+            'Bartlett1932 = demos.bartlett1932.experiment:Bartlett1932',
```

Comment:
Is the idea that we'll add a line here for each of the demos that we want to make accessible through `dallinger.experiments`?